test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -3868,6 +3868,1794 @@ public void EntryPointHashJoinCountTable()
                 });
         }
 
+        [Fact]
+        public void TestSimpleExperiment()
+        {
+            var dataPath = GetDataPath("adult.tiny.with-schema.txt");
+            string inputGraph = @"{
+                'Nodes':
+                [{
```

Comment:
am curious, how does one construct the json graph like the ones you used in these tests ?

Reply:
json is my second language in case you didn't know :) jokes aside, the old CSharpAPI basically constructs a json graph underneath and then passes it to the graph runner that executes the pipeline by calling into entrypoints. I stepped into the debugger and got the graph and replaced the pipeline with it while keeping rest of the test the same. I used graph runner to run the graph and get the idataview object that the test was using.